### ISSUE TITLE
Battery State not updating on Movesense

### ISSUE THREAD

**Issue by Goubeast** [cph-cachet/carp.sensing-flutter#459]
The battery is always at 80% for the movesense device - see image below

![WhatsApp Image 2024-12-18 at 14 40 12](https://github.com/user-attachments/assets/1f3abe3d-63df-43b7-91ce-14271f0bec5e)

This may be because  '_addStateSubscription' is not implemented for battery in 'enum MovesenseState' - line 197, carp_movesense_package/lib/movesense_probes.dart

**Comment by iarata** [comment]
I think from Movesense we could only get either "OK" (0) or "LOW" (1) for battery status and in the code it was converted to either 80% (OK) or 10% (LOW). See [`carp_movesense_packag/lib/movesense_device_manager.dart`](https://github.com/cph-cachet/carp.sensing-flutter/blob/6903940e2b42e99c74c4dfe26af3af15b8c2106c/packages/carp_movesense_package/lib/movesense_device_manager.dart#L222)

**Comment by bardram** [comment]
> I think from Movesense we could only get either "OK" (0) or "LOW" (1) for battery status and in the code it was converted to either 80% (OK) or 10% (LOW). See [`carp_movesense_packag/lib/movesense_device_manager.dart`](https://github.com/cph-cachet/carp.sensing-flutter/blob/6903940e2b42e99c74c4dfe26af3af15b8c2106c/packages/carp_movesense_package/lib/movesense_device_manager.dart#L222)

Yes - this is true. So, as long as the battery stage is OK, it reads 80%.

We might need to update the UX design to reflect that the battery state can be other than a percentage. @urganerea
